broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -32,6 +35,72 @@ const CloudControllerError = {
   }
 };
 
+
+function getResourceAnnotationStatus(resourceType, resourceId, guid, start_state, started_at) {
+  logger.info(`getResourceAnnotationStatus is called`);
```

Comment:
this log is useless right ? better to have more info.

Reply:
sure